discovery-wrapper.py

```diff
@@ -188,13 +190,16 @@ def memc_touch(key, time):
     Take the amount of threads we want to run in parallel from the commandline
     if None are given or the argument was garbage, fall back to default of 16
 """
-try:
-    amount_of_workers = int(sys.argv[1])
-    if amount_of_workers == 0:
-        print "ERROR: 0 threads is not a valid value"
-        sys.exit(2)
-except:
-    amount_of_workers = 1
+parser = argparse.ArgumentParser(description='Spawn multiple discovery.php processes in parallel.',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-d', '--debug', action='store_true', default=False,
+                    help="Enable debug output.")
```

Comment:
We should definitely warn here that this could lead to a large increase in disk usage if left enabled.